compiler/cpp/src/util/JSONParser.cpp

```diff
@@ -1,10 +1,14 @@
-//
-// Created by Marc on 24.05.2021.
-//
+// Copyright (c) 2021 Compose Generator Contributors. All rights reserved.
 
 #include "JSONParser.h"
 
+#include <utility>
+
 json JSONParser::getJSONValueFromKey(const std::unique_ptr<KeyExprAST>& key) {
+    return getJSONValueFromKey(data, key);
+}
+
+json JSONParser::getJSONValueFromKey(const json data, const std::unique_ptr<KeyExprAST>& key) {
     json dataTmp = data;
     for (const std::unique_ptr<IdentifierExprAST>& identifier : key->getIdentifiers()) {
         std::string identifierName = identifier->getName();
```

Comment:
you can move `identifierIndex` down to line 20, right? (sorry GitHub doesn't let me choose the line I intend as you didn't change it in this PR)

Reply:
Yes, alway good to fail faster when having an error 😄

Reply:
Fixed it.